vrk-kpa/suomidigi
# SUOPA-830 Error page

To test:
- `make fresh drush-cr`
- Go a broken url like https://suomidigi.docker.sh/asdlfjkasdgkl and check that 404-error page is like in proto: https://xd.adobe.com/view/d3c60a1d-1d76-4a8b-4797-030e9c2e717f-a5f7/screen/0ed181b4-838d-47fe-a4e6-d9f44387ebc4/Suomidigi-2020-404-XL
